wordpress-mobile/WordPress-Android
# [Reader] Use `"follow"` for tag strings instead of `"subscribe"`

Fixes #20297 

-----

## To Test:

1. Launch Jetpack and login
2. Do a general smoke test of Reader screens
3. All tag related strings should be using `"follow"` instead of `"subscribe"`
-----

## Regression Notes

1. Potential unintended areas of impact

    - None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

3. What automated tests I added (or what prevented me from doing so)

    --

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
